.github/workflows/proto-lint.yml

```diff
@@ -0,0 +1,37 @@
+name: proto-checks
+'on':
+  - push
+  - pull_request
+jobs:
+  proto-lint:
+    name: proto-checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.18.x
+
+      - name: Setup build depends
+        run: |
+         go get github.com/grpc-ecosystem/grpc-gateway/v2/internal/descriptor@v2.10.0
+         go get google.golang.org/grpc/cmd/protoc-gen-go-grpc
+         go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-grpc-gateway
+         go install github.com/grpc-ecosystem/grpc-gateway/v2/protoc-gen-openapiv2
+         go install google.golang.org/protobuf/cmd/protoc-gen-go
+         go install google.golang.org/grpc/cmd/protoc-gen-go-grpc
```

Comment:
Should these be part of the readme?

Reply:
The core ones required are already mentioned in: https://github.com/thrasher-corp/gocryptotrader/tree/master/gctrpc#installation and the same as the official docs: https://github.com/grpc-ecosystem/grpc-gateway#installation

Issuing these commands will report any dependant ones like the descriptor which can be left to the user to resolve and can be updated periodically